<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Available</class>
 <widget class="QDialog" name="Available">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget_available">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_add">
      <attribute name="icon">
       <iconset resource="../../../resources/icons.qrc">
        <normaloff>:/icons/icons/file-earmark-plus.svg</normaloff>:/icons/icons/file-earmark-plus.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>ADD</string>
      </attribute>
      <layout class="QGridLayout" name="add_layout"/>
     </widget>
     <widget class="QWidget" name="tab_list">
      <attribute name="icon">
       <iconset resource="../../../resources/icons.qrc">
        <normaloff>:/icons/icons/card-list.svg</normaloff>:/icons/icons/card-list.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>LIST</string>
      </attribute>
      <layout class="QGridLayout" name="list_layout"/>
     </widget>
     <widget class="QWidget" name="tab_update">
      <attribute name="icon">
       <iconset resource="../../../resources/icons.qrc">
        <normaloff>:/icons/icons/arrow-up-circle.svg</normaloff>:/icons/icons/arrow-up-circle.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>UPDATE</string>
      </attribute>
      <layout class="QGridLayout" name="update_layout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
